monkey/monkey_island/cc/ui/src/components/utils/PasswordBasedEncryptor.tsx

```diff
@@ -0,0 +1,11 @@
+import AES from 'crypto-js/aes';
+import Utf8 from 'crypto-js/enc-utf8';
+
+export function encryptText(content: string, password: string): string {
+  return AES.encrypt(content, password).toString();
```

Comment:
`AES.encrypt(content, password)` already returns base64, maybe the `.toString();` is not required, but that's how it's shown in the documentation, I guess it doesn't bother me and explains that our end result is a string

Reply:
It might pay to be more specific and specify the encoder:

![image](https://user-images.githubusercontent.com/19957806/176668279-0eb05381-950c-4b6e-895f-d8cb8c92705e.png)